backend/elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/types/predicate/impl/ElasticsearchStandardMatchPredicateBuilder.java

```diff
@@ -53,12 +53,12 @@
 	ElasticsearchStandardMatchPredicateBuilder(ElasticsearchSearchContext searchContext,
 			String absoluteFieldPath,
 			ToDocumentFieldValueConverter<?, ? extends F> converter, ToDocumentFieldValueConverter<F, ? extends F> rawConverter,
-			ElasticsearchDslConverterHandler dslConverterHandler, ElasticsearchFieldCodec<F> codec) {
+			ElasticsearchConverterCompatibilityChecker compatibilityChecker, ElasticsearchFieldCodec<F> codec) {
 		this.searchContext = searchContext;
 		this.absoluteFieldPath = absoluteFieldPath;
 		this.converter = converter;
 		this.rawConverter = rawConverter;
-		this.dslConverterHandler = dslConverterHandler;
+		this.compatibilityChecker = compatibilityChecker;
```

Comment:
Same here, I think naming it `converterCompatibilityChecker`, or `converterChecker` if you want a short name, would be better. `compatibilityChecker` makes it seem like we're checking the compatibility of the codec, too.

Reply:
Ok, then I'll rename all parameter, variable and field named `compatibilityChecker` in `converterChecker`.